src/node_api.cc

```diff
@@ -156,14 +156,20 @@ class HandleScopeWrapper {
 // across different versions.
 class EscapableHandleScopeWrapper {
  public:
-  explicit EscapableHandleScopeWrapper(v8::Isolate* isolate) : scope(isolate) {}
+  explicit EscapableHandleScopeWrapper(v8::Isolate* isolate) :
+    scope(isolate), escapeCalled(false) {}
+  bool escapeAllreadyCalled(void) {
+    return escapeCalled;
+  }
   template <typename T>
   v8::Local<T> Escape(v8::Local<T> handle) {
+    escapeCalled = true;
     return scope.Escape(handle);
   }
 
  private:
   v8::EscapableHandleScope scope;
+  bool escapeCalled;
```

Comment:
Private members should have an underscore prefix. At least that is the style used in other classes in this file.